node-services/user-service/controllers/validateParameters.js

```diff
@@ -0,0 +1,16 @@
+const validate = {
+  validateEmail: (email) => {
+    const regex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
I'll assume this is correct, I can't possibly read this. Side note, read about 'catastrophic backtracking regex'